AR-builder.groovy

```diff
@@ -42,6 +47,33 @@ def postGithubComment(String changeId, String body) {
 
 }
 
+def postGithubApkToRelease(String flavor, String buildType) {
+    def apks = findFiles(glob: "app/build/outputs/apk/${flavor.toLowerCase()}/${buildType.toLowerCase()}/com.wire.android-*.apk")
+    if (apks.size() > 0) {
+        echo 'Attaching APK to Github Release for tag: ' + env.SOURCE_BRANCH
+        def fileApk = apks[0]
+        // note: apk name value rename to comply with github releases assets names requirements
+        // https://docs.github.com/en/rest/releases/assets?apiVersion=2022-11-28#upload-a-release-asset
+        def filename = fileApk.getName().replaceAll("\\(", "_").replaceAll("\\)", "_")
+
+        // building headers request
+        def acceptHeader = shellQuote("Accept: application/vnd.github.v3+json")
+        def contentTypeHeader = shellQuote("Content-Type: application/octet-stream")
+        def authHeader = shellQuote("Authorization: token ${env.GITHUB_API_TOKEN}")
+
+        // building gh api request
+        def apiUrl = shellQuote("https://api.github.com/repos/wireapp/wire-android/releases/latest")
+        def releaseId = sh(script: "curl -s ${apiUrl} | grep -m 1 \"id.:\" | grep -w id | tr : = | tr -cd '[[:alnum:]]=' | cut -d'=' -f2", returnStdout: true).trim()
```

Comment:
In an ideal world, this can be simplified if `./jq` would be available. 
But `grep`, `tr` and `cut` can also manage to do it.